pkg/ccl/backupccl/schedule_exec.go

```diff
@@ -164,23 +166,25 @@ func (e *scheduledBackupExecutor) NotifyJobTermination(
 	txn isql.Txn,
 	jobID jobspb.JobID,
 	jobStatus jobs.Status,
+	jobErr error,
 	details jobspb.Details,
 	env scheduledjobs.JobSchedulerEnv,
-	schedule *jobs.ScheduledJob,
+	sj *jobs.ScheduledJob,
 ) error {
 	if jobStatus == jobs.StatusSucceeded {
 		e.metrics.NumSucceeded.Inc(1)
-		log.Infof(ctx, "backup job %d scheduled by %d succeeded", jobID, schedule.ScheduleID())
```

Comment:
this implementation change makes this function a bit harder to understand: we return an error to the user, but don't use the newly passed in the `jobErr`. I wonder if we can avoid this big change by merely incrementing this metric in the backup job itself, as I propose.

Reply:
same as before, I think it's good to notify with the error anyways since it's something that could be useful for jobs with a failed status.

Reply:
hrm: I wonder if there's a way to add this feature without complicating the error handling here. It doesn't sit well with me that we have `jobErr` passed in and a different error generated from the `jobStatus` returned to the user. I'm going to ping job peeps to take a look at this.